components/Accordion/src/AccordionToggle.js

```diff
@@ -19,20 +24,30 @@ const AccordionToggle = (className = 'denhaag-accordion__title') => {
 
       if (expanded === 'false') {
         // Return correct height to show closing animation
-        setAccordionHeight(parent, accordionHeight);
+        setAccordionHeight(wrapper, accordionHeight);
 
         setTimeout(() => {
-          parent.classList.toggle('denhaag-accordion__container--open');
+          wrapper.classList.toggle('denhaag-accordion__container--open');
         }, 50);
+
```

Comment:
Ik vraag me af, is die setTimeout echt nodig?

Reply:
Ik dacht eerst ook van niet, zolang die functie maar als laatste wordt aangeroepen, maar zonder timeout klapt hij soms in een keer open zonder animatie. Misschien omdat er toch nog tijd overeen gaat voordat de height in de html is geupdate?

Reply:
Dat zou gebeuren als er geen `DOMContentLoaded` op zit, en anders een `load` ?

Ben altijd huiverig met de setTimeouts :P